### PR TITLE
Bug 1861242 - Strings for add-ons general availability Nimbus message.

### PR DESCRIPTION
**Pull request by Amejia481**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1861242

### CODE REVIEW

File: fenix/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -662,6 +662,14 @@
     <!-- Wallpaper onboarding dialog learn more button text. The button navigates to the wallpaper settings screen. -->
     <string name="wallpapers_onboarding_dialog_explore_more_button_text">Explore more wallpapers</string>
 
+    <!-- Add-ons general availability nimbus message-->
+    <!-- Title of the Nimbus message for add-ons general availability-->
+    <string name="addon_ga_message_title" tools:ignore="UnusedResources">New add-ons now available</string>
+    <!-- body of the Nimbus message for add-ons general availability. 'Firefox' intentionally hardcoded here-->
```
Comment (sarah541) [inline fenix/app/src/main/res/values/strings.xml:668]:
```suggestion
    <!-- Body of the Nimbus message for add-ons general availability. 'Firefox' intentionally hardcoded here-->
```

File: fenix/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -662,6 +662,14 @@
...
+    <string name="addon_ga_message_body" tools:ignore="UnusedResources">Check out 100+ new extensions that let you make Firefox your own.</string>
+    <!-- Button text of the Nimbus message for add-ons general availability. -->
+    <string name="addon_ga_message_button" tools:ignore="UnusedResources">Explore add-ons.</string>
```
Comment (sarah541) [inline fenix/app/src/main/res/values/strings.xml:671]:
```suggestion
    <string name="addon_ga_message_button" tools:ignore="UnusedResources">Explore add-ons</string>
```

**Review by sarah541 — CHANGES_REQUESTED** [review]

File: fenix/app/src/main/res/values/strings.xml (hunk 2 above)
Comment (Amejia481) [reply fenix/app/src/main/res/values/strings.xml]: Addressed!

File: fenix/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Amejia481) [reply fenix/app/src/main/res/values/strings.xml]: Addressed

**Comment by Amejia481** [comment]
Thanks for the review!
I updated the PR with your suggestions :)

**Review by sarah541 — APPROVED** [review]

**Comment by Amejia481** [comment]
@Mergifyio backport releases_v120

**Pull request merged**